shared/src/Datadog.Trace.ClrProfiler.Native/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required (VERSION 3.13.4)
```

Comment:
I think this was a side effect of a massive replace, this line should probably be reverted ?